vendor/blanket-require.js

```diff
@@ -50,6 +50,10 @@ var shouldExclude = function(moduleName) {
       return true;
     }
 
+    if ( moduleName === blanket.options('modulePrefix') ) {
```

Comment:
I believe in that case the module would look like

define('sl-ember-components/index', ['sl-ember-components/index',

so it wouldn't be excluded.

I'm trying to get my head back into this project space after a while away, so correct me if I'm wrong